### PR TITLE
Rename manage/release to activate/deactivate.

### PR DESCRIPTION
**Pull request by sjoerdtalsma**
The terms _activate_ and _deactivate_ seemed more appropriate than _manage_ and _release_, as became apparent during the Distributed Tracing workshop in Berlin.

I think it is wise to tackle unclear naming while the library still is in the 'work in progress' stage.

### CODE REVIEW

**Comment by sjoerdtalsma** [comment]
Tagging @adriancole and @wu-sheng as I can't seem to add you as official 'reviewers' for this PR, but I'm curious about your opinions.

Btw, no problem if you don't have any time..

**Comment by sjoerdtalsma** [comment]
@objectiser I guess this would have impact on your code?
Interested in all opinions on this, I just t.ry to take advantage of what felt like consensus on monday 😉 .

**Review by bhs — APPROVED** [review]
LGTM!

**Comment by objectiser** [comment]
@sjoerdtalsma LGTM - once merged if you can release then I'll update my code.

**Review by wu-sheng — APPROVED** [review]
LGTM, these two names are better.😄

**Comment by sjoerdtalsma** [comment]
@objectiser Merged and release 0.0.3 was created. 
It may take a while before maven central mirrors pick it up though.

**Comment by objectiser** [comment]
@sjoerdtalsma Thanks

**Pull request merged**
